templates/zerver/emails/email.css

```diff
@@ -84,65 +86,52 @@ table td {
     margin-top: 20px;
 }
 
-.footer td,
-.footer span {
+.content-block,
+.apple-link {
     color: #5f5ec7;
     font-size: 12px;
     text-align: center;
 }
 
-h1,
-h2,
-h3,
-h4 {
-    color: #4e4d4d !important;
-    font-family: sans-serif;
-    font-weight: 600;
-    line-height: 1.4;
-    margin-bottom: 30px;
-}
-
-h1 {
-    font-size: 35px;
-    font-weight: 300;
-    text-align: center;
-    text-transform: capitalize;
-}
-
-h2 {
-    font-size: 24px;
-    margin-top: 20px;
-    margin-bottom: 10px;
-}
-
-ul,
-ol {
+.content ul,
+.content ol {
     /* Override excessive indentation seen in Gmail. */
     padding-left: 5px;
 }
 
-p,
-ul,
-ol {
+.body p,
+.digest_paragraph,
+.content ul,
+.content ol {
     font-family: sans-serif;
     font-size: 14px;
     font-weight: normal;
     margin: 10px 0;
 }
 
-p li,
-ul li,
-ol li {
+.body p li,
```

Comment:
We don't use the `.digest_paragraph` class here, as it does not contain any <li> elements.